.buildkite/pipeline.yml

```diff
@@ -25,3 +25,52 @@ steps:
       - build-apple
     artifact_paths:
       - "screenshots/**/*.png"
+
+  - label: ":bazel: build"
+    key: bazel-build
+    soft_fail: true
```

Comment:
```suggestion
```

Reply:
I'd suggest we don't use soft_fail steps for this, I think Bazel config should be pretty stable once it lands.

Reply:
I wanted to allow this to fail at the moment as we don't want to force bazel to be successful (yet). Especially as it's not working right now anyway 😂